config_example.toml

```diff
@@ -0,0 +1,144 @@
+# The global section has parameters that apply globally to the relayer operation.
+[global]
+
+# Specify the verbosity for the relayer logging output. Default: 'info'
+# Valid options are 'error', 'warn', 'info', 'debug', 'trace'.
+log_level = 'trace'
+
+
```

Comment:
Should add our new config (delay, and ignore ack)?

Reply:
There are ignore ack here, but yeah, should add delay here.